<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiPotentiostat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="osc"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionCalibration"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuDevice"/>
   <addaction name="menuCalibration"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Setup</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Work electrode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,0,0,1,1,1,1,1,5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Voltage gain</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="workVoltGain">
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>x1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Current gain A</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="workCurrGainA">
            <item>
             <property name="text">
              <string>x1k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x10k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x1000k</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Current gain B</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="workCurrGainB">
            <item>
             <property name="text">
              <string>x10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x1000</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="workGainInfo">
          <property name="text">
           <string>Total range:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="workVolt"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>[mV]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Sweep to</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="workSweepTo"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>[mV]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="sweepWork">
            <property name="text">
             <string>Sweep</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="pullProbe">
            <property name="text">
             <string>Pull probe when sweep</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>36</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Probe</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,0,0,1,1,1,1,1,5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Voltage gain</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="probeVoltGain">
            <item>
             <property name="text">
              <string>x1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Current gain A</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="probeCurrGainA">
            <item>
             <property name="text">
              <string>x1k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x10k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100k</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x1000k</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Current gain B</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="probeCurrGainB">
            <item>
             <property name="text">
              <string>x10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x1000</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="probeGainInfo">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="probeVolt"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>[mV]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Sweep to</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="probeSweepTo"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>[mV]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="sweepProbe">
            <property name="text">
             <string>Sweep</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="pullWork">
            <property name="text">
             <string>Pull work when sweep</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>37</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open/Reopen</string>
   </property>
  </action>
  <action name="actionDC_Volt">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/dc.png</normaloff>:/images/dc.png</iconset>
   </property>
   <property name="text">
    <string>DC Volt</string>
   </property>
   <property name="toolTip">
    <string>DC voltage controls</string>
   </property>
  </action>
  <action name="actionSingle_pulse">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/pulse.png</normaloff>:/images/pulse.png</iconset>
   </property>
   <property name="text">
    <string>Single pulse</string>
   </property>
   <property name="toolTip">
    <string>Single voltage pulse controls</string>
   </property>
  </action>
  <action name="actionMeandr">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/meandr.png</normaloff>:/images/meandr.png</iconset>
   </property>
   <property name="text">
    <string>Meandr</string>
   </property>
   <property name="toolTip">
    <string>Voltage meandr controls</string>
   </property>
  </action>
  <action name="actionShort_relay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/short-relay.png</normaloff>:/images/short-relay.png</iconset>
   </property>
   <property name="text">
    <string>Short relay</string>
   </property>
  </action>
  <action name="actionOut_relay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/out-relay.png</normaloff>:/images/out-relay.png</iconset>
   </property>
   <property name="text">
    <string>Out relay</string>
   </property>
   <property name="toolTip">
    <string>Out relay</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
  <action name="actionSweep">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/sweep.png</normaloff>:/images/sweep.png</iconset>
   </property>
   <property name="text">
    <string>Sweep</string>
   </property>
  </action>
  <action name="actionExecLua">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/exec.png</normaloff>:/images/exec.png</iconset>
   </property>
   <property name="text">
    <string>execLua</string>
   </property>
   <property name="toolTip">
    <string>Execute Lua script</string>
   </property>
  </action>
  <action name="actionStopLua">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>stopLua</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
